src/components/lists/ListDetail.js

```diff
@@ -31,14 +35,39 @@ class ListDetail extends Component {
     });
   }
 
+  onToggleList(id) {
+    this.props.ToggleList({
+      variables: { id },
+      refetchQueries: [{
+        query: FetchList,
```

Comment:
the real reason ```onToggleList()``` isn't in the ListToggle component is cause the refetch query calls different data (refetch query can be passed in... refactor this)